Tests/KlaviyoSwiftTests/KlaviyoStateTests.swift

```diff
@@ -113,33 +113,40 @@ final class KlaviyoStateTests: XCTestCase {
         environment.fileClient.fileExists = { _ in
             true
         }
-        environment.data = { _ in
-            try! JSONEncoder().encode(KlaviyoState(apiKey: "foo", anonymousId: environment.analytics.uuid().uuidString, queue: [], requestsInFlight: []))
+        environment.dataFromUrl = { _ in
+            try! JSONEncoder().encode(KlaviyoState(
+                apiKey: "foo",
+                anonymousId: environment.uuid().uuidString,
+                queue: [],
+                requestsInFlight: []))
         }
-        environment.analytics.decoder = DataDecoder(jsonDecoder: KlaviyoEnvironment.decoder)
+//        environment.decoder = DataDecoder(jsonDecoder: KlaviyoEnvironment.decoder)
```

Comment:
Is this intentional?

Reply:
Sorta! it was there before and the test passed w/o it so kept it around to make sure I'm not missing something. Will take a second look and remove it if it's absolutely not needed.

Reply:
Didn't look like it was needed. Removed it.